src/printer.js

```diff
@@ -69,7 +69,7 @@ function printTopLevelParts(node, path, opts, print) {
   }
 
   const docs = flatten([parts.frontmatter, ...parseSortOrder(opts.astroSortOrder).map((p) => parts[p])]).filter((doc) => '' !== doc);
-  return docs;
+  return group([join(softline, docs)]);
```

Comment:
The test suite seems to fail with this—it inserts a double-line between `---` and the first line of HTML